docs/source/release/v6.1.0/mantidworkbench.rst

```diff
@@ -8,53 +8,66 @@ Mantid Workbench Changes
 New and Improved
 ----------------
 
-- Added Floating/On Top setting for all the windows that are opened by workbench (plots, interfaces, etc.)
-- New plot interactions: Double click a legend to hide it, double click a curve to open it in the plot config dialog.
-- It is now possible to overplot bin data from the matrix workspace view.
+General
+#######
+- Added setting to allow user to select Floating/On Top behaviour for all the windows that are opened by workbench (plots, interfaces, etc.). For more detail see ref:`Window Behaviour in Workbench<WorkbenchWindowBehavior>`.
```

Comment:
I think this is one of the big Core changes, and it's worth users knowing that they can chose this behaviour of interfaces. It's particularly important on say Ubuntu, where a user may choose `Floating` to allow the minimise/maximise buttons. This is all explained [here](https://docs.mantidproject.org/nightly/workbench/windowbehaviour.html) so please add a link to it with 
```
ref:`Whatever words you want<WorkbenchWindowBehavior>`
```
We could maybe even make a GIF related to this (they work on the forum, maybe on the docs?) or just an image of the new setting that can be changed. Feel free to poke me about this on Monday!

Reply:
GIF's work in the docs. We have them in the muon tutorials

Reply:
https://github.com/mantidproject/mantid/blob/master/docs/source/tutorials/muon_GUI_course/the_tabs_grouping.rst

Reply:
Thanks @DanielMurphy22 - I added the GIF you made!